packages/client/hmi-client/src/components/model/model-parts/tera-model-part-entry.vue

```diff
@@ -135,63 +138,25 @@ if (props.item.description) showDescription.value = true;
 
 <style scoped>
 section {
-	display: grid;
-	grid-template-areas:
-		'symbol name unit . concept'
-		'expression expression expression expression expression'
-		'description description description description description';
-	grid-template-columns: max-content max-content max-content auto max-content;
```

Comment:
Why not just adding a `grid-template-rows` to make the expression as tall as needed?